Formula/svt-av1.rb

```diff
@@ -0,0 +1,32 @@
+class SvtAv1 < Formula
+  desc "AV1 encoder"
+  homepage "https://gitlab.com/AOMediaCodec/SVT-AV1"
+  url "https://gitlab.com/AOMediaCodec/SVT-AV1/-/archive/v1.1.0/SVT-AV1-v1.1.0.tar.gz"
+  sha256 "1c211b944ac83ef013fe91aee96c01289da4e3762c1e2f265449f3a964f8e4bc"
+  license "BSD-3-Clause"
+  head "https://gitlab.com/AOMediaCodec/SVT-AV1.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "make" => :build
+  depends_on "yasm" => :build
+
+  resource("testvideo") do
```

Comment:
```suggestion
  resource("homebrew-testvideo") do
```
This marks it as internal to the formula so `brew bump-formula-pr` won't complain that it might need updating.

You'll need to rename the bit in the `test` block too.

Reply:
Thanks, PR updated according to suggestion.